Cargo.lock

```diff
@@ -101,6 +101,7 @@ dependencies = [
  "globset",
  "grep-cli",
  "lazy_static",
+ "lazycell",
```

Comment:
syntect already directly depends on this crate, so it is not a big deal for us to also start doing that